prefect_dbt/cli/models.py

```diff
@@ -0,0 +1,98 @@
+"""Module containing models for configs"""
```

Comment:
My opinion would be to call this module `prefect_dbt.cli.configs`.  I think that's more descriptive. We may even want to make submodules since there will be quite a few of these (e.g. `prefect_dbt.cli.configs.snowflake`)